Xamarin.Forms.Platform.iOS/Renderers/EditorRenderer.cs

```diff
@@ -152,6 +154,8 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdatePlaceholderColor();
 			else if (e.PropertyName == Editor.AutoSizeProperty.PropertyName)
 				UpdateAutoSizeOption();
+			else if (e.PropertyName == Xamarin.Forms.InputView.IsReadOnlyProperty.PropertyName)
```

Comment:
It looks like this code will never hit because of the previous condition on [line 137](https://github.com/xamarin/Xamarin.Forms/pull/1972/files#diff-71148fd2814c3d17185db42cc58913ffR137).

Reply:
You are right I will take a look, also there is some code missing, from after my previous rebase...